### ISSUE TITLE
Production Puzzles

### ISSUE THREAD

**Issue by flexagontnt** [gtw123/OpusSolver#6]
Can you program the but to do production puzzles? If so, how long will it take to program it?

**Comment by gtw123** [comment]
The current "algorithm" it uses to create a solution only works because there is unlimited space available to place all the parts. It wouldn't work for production puzzles and I have no idea how to make one that would...

**Comment by ShylokVakarian** [comment]
You could always program the bot to recognize which level it is and just reconstruct a pre-existing solution.  There's not a whole lot of production puzzle levels, 15 max.

**Comment by gtw123** [comment]
Good point, but I think someone made a separate puzzle editor that lets you make custom production puzzles. I think I saw some in the Steam workshop at some point.

**Comment by ShylokVakarian** [comment]
True, you could never account for those, but you CAN account for the ones already in the game.  Limited support for production puzzles is still support.
